dev/puppetserver.conf

```diff
@@ -83,6 +83,10 @@ jruby-puppet: {
     # (optional) whether to use the environment class cache. If unspecified
     # defaults to false
     environment-class-cache-enabled: true
+
+    # (optional) Whether or not to implement a lookup recorder during compilation to track
+    # what keys are looked up and how often
```

Comment:
should probably use less implementation specific language and say something about what the end result is